riscv/processor.cc

```diff
@@ -614,7 +622,7 @@ void processor_t::set_csr(int which, reg_t val)
       break;
     case CSR_SSTATUS: {
       reg_t mask = SSTATUS_SIE | SSTATUS_SPIE | SSTATUS_SPP | SSTATUS_FS
-                 | SSTATUS_XS | SSTATUS_SUM | SSTATUS_MXR;
+                 | SSTATUS_XS | SSTATUS_SUM | SSTATUS_MXR | SSTATUS_VS;
```

Comment:
I think this should be `(supports_extension('V') ? SSTATUS_VS : 0` -- in other words, the field doesn't exist without V extension.  This is different than FS, but I think that's what we need to do since this is a new extension.

Reply:
It is reasonable and has been changed in new PR